### ISSUE TITLE
obfuscate main program

### ISSUE THREAD

**Issue by xKajika** [edvin/fxlauncher#167]
Hello,

I want to use Proguard to obfuscate my main application after building the Project.
The problem is, the launcher will download every restart the obfuscated application, maybe because the app.xml in the launcher.

there is a way to fix that? Can i create or replace the app.xml in the launcher myself?

**Comment by edvin** [comment]
Yes, it's just a zip file, so you can open it and swap out the app.xml, or use the jar command to replace it :)

**Comment by xKajika** [comment]
Ok, i just have to generate a new checksum, with ur alogirithm, and replace it in the app.xml.

That will fix it?

**Comment by edvin** [comment]
You can do that, but you could also just make sure the obfuscation happens before you bundle the app.xml file in the launcher, so you don't need to fiddle with this at all.

**Comment by xKajika** [comment]
I have no idea how to do this, acutally i obfuscate with the Proguard GUI. So i have to build, take the jar and obfuscate.

**Comment by edvin** [comment]
Just do that before you run the create-manifest and embed-manifest-in-launcher step. Refer to the maven or gradle example project for more info and examples :)

**Comment by xKajika** [comment]
i´ll try it, thank you for reply!
